docs/user/library/jdbc/index.rst

```diff
@@ -39,6 +39,7 @@ DataStore plugins:
    
    db2
    h2
+   hana
```

Comment:
Do you need to change the hana documentation page at all?

Reply:
As far as I can tell no further change is necessary. The documentation page is already there (https://docs.geotools.org/latest/userguide/library/jdbc/hana.html) and all the documentation is still up-to-date. Just this file needs an update to move the plugin from the list of unsupported plugins to the supported ones.

Or am I forgetting something?

Reply:
Had a look, I like the utility to load additional EPSG definitions.

The pom contains instructions on locally adding the jdbc driver, yep I think we are good.